Anglish for Rockhill Ancestry

FewOldTunes opened the pull request:
Some people may ask a simple question; "Why?"

And to which, I believe it is funny.

Abraxas was the one who suggested it aswell...

Brief Summary: Adds a Anglish accent to the Rockhill ancestry in humans. It's similar to Grenzelhoft where it is still understandable just slightly annoying.

Real-MAGNUM commented:
Testmerging.

ophaq commented:
> Some people may ask a simple question; "Why?"
> 
> And to which, I believe it is funny.
> 
> Abraxas was the one who suggested it aswell...
> 
> Brief Summary: Adds a Anglish accent to the Rockhill ancestry in humans. It's similar to Grenzelhoft where it is still understandable just slightly annoying.

I've got a few ideas too. Could you we make use the of the word replacement mechanic and force accents and some words, like day = dae, no = nay, yes = aye, and make dark elfs have french accents and dwarves have scottish accents? Also it would be funny if all 'S' letters with the medieval 'f' letter like in medieval scriptures.

BlueMemesauce commented:
Why does this apply to the rockhill ancestry? the game literallly takes place in rockhill there shouldn't be any accent for it

wow50000 commented:
why the fuck is court physician "hof leech"

Real-MAGNUM commented:
untestmerged

YosemiteYam commented:
No, just no. Rockhill is the native populace, why would they have an accent? I choose rockhill for my ancestry, especially for court/king roles so I don't have to deal with dumb fucking accent bullshit. Stop forcing accents down our throats that would turn NATIVE WORDS and the OFFICIAL TERMS OF THE KINGDOM into GIBBERISH by its own native people.

FewOldTunes commented:
> why is court physician "hof leech" not to mention why does it replace bog guard with mire ward, just please remove any of the words that are actual jobs or at least make them resemble the original name also no being nag is doing some weird fuckery like nagble or something now that im looking through the words most of them are actually pretty good given how these three are the only ones i can find

I fixed this now; it was a bit overlooked on my part majorly to include any 2-3 letter words. I removed a good 30-ish words now that made no sense.

FewOldTunes commented:
> No, just no. Rockhill is the native populace, why would they have an accent? I choose rockhill for my ancestry, especially for court/king roles so I don't have to deal with dumb fucking accent bullshit. Stop forcing accents down our throats that would turn NATIVE WORDS and the OFFICIAL TERMS OF THE KINGDOM into GIBBERISH by its own native people.

Do you think it'd be better to apply it to Hammerhold? Your argument makes sense; and I'm willing to give it a try to be honest.

FewOldTunes commented:
Alright, I'm changing it tto Hammerhold, it makes more sense there after further internal discussion.

ChungusGamer666 commented:
It does, make more sense, for it to be hammerhold. For Rockhill a very mild british accent would make more sense.

ChungusGamer666 commented:
This makes sense for hammerhold because they are in fact, if you think about it, our equivalent to anglo-saxons as they were when invading britain
